tests/sentry/workflow_engine/models/test_action.py

```diff
@@ -0,0 +1,68 @@
+from unittest.mock import Mock, patch
+
+import pytest
+from django.test import TestCase
+
+from sentry.eventstore.models import GroupEvent
+from sentry.utils.registry import NoRegistrationExistsError
+from sentry.workflow_engine.models import Action
+from sentry.workflow_engine.types import ActionHandler
+
+
+class TestAction(TestCase):
+    def setUp(self):
+        self.mock_event = Mock(spec=GroupEvent)
+        self.mock_detector = Mock(name="detector")
+        self.action = Action(type=Action.Type.NOTIFICATION)
+
+    def test_get_handler_notification_type(self):
+        with patch("sentry.workflow_engine.registry.action_handler_registry.get") as mock_get:
+            mock_handler = Mock(spec=ActionHandler)
+            mock_get.return_value = mock_handler
+
+            handler = self.action.get_handler()
+
+            mock_get.assert_called_once_with(Action.Type.NOTIFICATION)
+            assert handler == mock_handler
+
+    def test_get_handler_webhook_type(self):
+        with patch("sentry.workflow_engine.registry.action_handler_registry.get") as mock_get:
+            mock_handler = Mock(spec=ActionHandler)
+            mock_get.return_value = mock_handler
+
+            handler = self.action.get_handler()
+
+            mock_get.assert_called_once_with(Action.Type.NOTIFICATION)
```

Comment:
This test and the one above it are identical except for the name, is this one supposed to use the webhook type?

Reply:
yes! (TY) i'm actually moving stuff around a little rn to make way for the `process_workflow_engine` [method](https://github.com/getsentry/sentry/pull/81645#discussion_r1870086196). will update.